libbeat/processors/add_docker_metadata/add_docker_metadata.go

```diff
@@ -65,6 +69,18 @@ type addDockerMetadata struct {
 
 // New constructs a new add_docker_metadata processor.
 func New(cfg *common.Config) (processors.Processor, error) {
+	// check if Docker is available in running environment
```

Comment:
Could you move the code to check if docker is available to a function? Then this comment will be probably unneeded :slightly_smiling_face:

Reply:
Sure! Nice catch!